redash/query_runner/databricks.py

```diff
@@ -157,10 +171,23 @@ def run_query(self, query, user):
 
                 rows = [
                     dict(zip((column["name"] for column in columns), row))
-                    for row in data
+                    for row in result_set
                 ]
 
                 data = {"columns": columns, "rows": rows}
+
+                if (
+                    len(result_set) >= settings.DATABRICKS_ROW_LIMIT
+                    and cursor.fetchone() is not None
+                ):
+                    # HACK: If this is a SQL endpoint (i.e. databricks_internal), "path" will exist
+                    endpoint_path = self.configuration.get("path", None)
+                    logger.warning(
+                        f"Truncated result set for query"
+                        f"path {endpoint_path}"
+                    )
```

Comment:
Let's simplify the log message:

```suggestion
                    logger.warning("Truncated result set.")
```

We will have to get the context from the surrounding log message related to the execution.